packages/react/src/select/index.tsx

```diff
@@ -1,5 +1,18 @@
 import { CdsSelect as SelectInput } from '@cds/core/select';
 import '@cds/core/select/register';
-import { createComponent } from '../converter/react-wrapper.js';
+import { createComponent } from '@lit-labs/react';
+import * as React from 'react';
 
-export const CdsSelect = createComponent('cds-select', SelectInput);
+export const CdsSelect = createComponent(React, 'cds-select', SelectInput);
+
+/**
+ * Setting displayName on the prototype is a workaround for now.
+ * See: https://github.com/lit/lit/issues/2154
+ * Follow: https://github.com/lit/lit/pull/2155
+ */
+CdsSelect.displayName = 'CdsSelect';
+
+// Setting react version has to be done at component level since we now consume the @lit-labs/react package
+if ((window as any).CDS && !(window as any).CDS._react.version) {
```

Comment:
Seems like we reuse the same code across all components. Let's do a util function and reuse it for flexibility